### PR TITLE
Issue #743: Change Edit link to Content Type Settings.

### LINKED ISSUE #743: [UX] Change "Edit" to "Content Type Settings" in Content Type Settings
The edit tab in the content type settings is not descriptive enough. Its easy to loose track of menu hierarchy while editing content types.

### PR DESCRIPTION
**Pull request by serundeputy**
This fixes: https://github.com/backdrop/backdrop-issues/issues/743

### CODE REVIEW

File: core/modules/node/node.module [hunk 1]
```diff
@@ -1625,7 +1625,7 @@ function node_menu() {
     'file' => 'node.types.inc',
   );
   $items['admin/structure/types/manage/%node_type/edit'] = array(
-    'title' => 'Edit',
+    'title' => 'Content Type Settings',
```
Comment (quicksketch) [inline core/modules/node/node.module]: In Backdrop/Drupal, "Sentence casing" is the standard for titles (as you can see in the other menu entries here). So this needs to be `Content type settings`. I've always thought this was a bit silly, but it's the standard and we should stick to it until it is changed everywhere.

**Comment by klonos** [comment]
@quicksketch you beat me to this one too. Was about to say the same thing.

**Comment by serundeputy** [comment]
thanks @klonos and @quicksketch adjusted and pushed back up.

**Comment by klonos** [comment]
How about this?:

```
- 'title' => 'Edit',
+ 'title' => %node_type . 'settings',
```

**Comment by serundeputy** [comment]
updated.

**Comment by serundeputy** [comment]
updated to `Settings`

**Comment by klonos** [comment]
Thanx Geoff and sorry for the constant corrections that you had to do for this PR. Change is pretty straight-forward. Looks good to go to me.

**Comment by quicksketch** [comment]
Ha, looks like "Configure" is the "final" suggestion here. When this is updated, could you squash all the commits together or rebase it to remove all the other attempts? 5 commits to change the same line clutters the commit history. Thanks!

**Comment by serundeputy** [comment]
yes Will do;

**Pull request closed without merging**